.changeset/olive-apricots-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/cli': patch
```

Comment:
is this only a patch? theoretically if someone was using an address as the HYP_KEY before, this would break right?

Reply:
You're right– the HYP_KEY will no longer support user addresses after this change. Updating to `minor`